imports/api/topics/votings/votings.js

```diff
@@ -55,7 +55,13 @@ const voteSchema = new SimpleSchema({
   procedure: { type: String, allowedValues: Topics.voteProcedureValues, autoform: _.extend({}, autoformOptions(Topics.voteProcedureValues, 'schemaVotings.vote.procedure.'), noUpdate) },
   effect: { type: String, allowedValues: Topics.voteEffectValues, autoform: _.extend({}, autoformOptions(currentUsersPossibleEffectValues, 'schemaVotings.vote.effect.'), noUpdate) },
   type: { type: String, allowedValues: Topics.voteTypeValues, autoform: _.extend({}, autoformOptions(Topics.voteTypeValues, 'schemaVotings.vote.type.'), noUpdate) },
-  choices: { type: Array, autoValue() { return Topics.voteTypes[this.field('vote.type').value].fixedChoices; } },
+  choices: { 
+    type: Array,
+    autoValue() { 
+      if (this.field('vote.type').value) return Topics.voteTypes[this.field('vote.type').value].fixedChoices; 
```

Comment:
Erre az if-re miert van szukseg?
A vote.type nem opcionalis mezo. Kotelezoen kene legyen erteke neki.
Elofordul hogy nincs?

Reply:
A szavazat szerkesztés autoformban disabled a vote.type mező, ezért nem kerül leküldésre submitkor.

Reply:
A disabled mezoket se kuldi le? Azt hittem csak az omit-oltat nem.
Mi mindent kell meg megtanulnom rola.
Akkor viszont surgosen be kell huzni ezt a branch-et a masterre. :)

Reply:
Nem sürgős, mert ezen a branchen objectify-oltad a vote.typesot, ez annak a fixe.